Sources/GoodExtensions/UIKit/UIButtonExtensions.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  UIButtonExtensions.swift
+//  
+//
+//  Created by Andrej Jasso on 12/02/2022.
+//
+
+import GRCompatible
+import CoreGraphics
+
+public extension GRActive where Base: UIButton {
+
+    func applyClickableAreaInsets(by value: CGFloat) {
```

Comment:
Toto sa mne osobne moc nepaci. Nepride mi to ako nieco univerzalne, hlavne z toho pohladu, ze reco musim zvacsovat edgeInset pre kazdu stranu rovnako. Skor mi to pride ako use case issue pre projekt ako nieco co budeme pozivat na viac projektoch. Co ty na to?

Reply:
Osobne by som to asi vyuzil par krat na kazdom projekte hlavne na navigacne buttony ale popravde je to odtialto tak som myslel ze existuje consenzus ze toto tam chalani chcu :D https://goodrequest.atlassian.net/wiki/spaces/IOS/pages/769102526152748/Best+practice 
Toto olek zbieral nejake bestpractices z projektov. Akoze prezil by som bez toho ale zasa vedel by som to pouzit.

Reply:
JA som to vzdy robil tak, že som zmenil constrainty. Čiže nikdy som nič takéto nepotreboval. Ako vravíš, je tu další problém a to že to bude  overlapovat ostatne komponenty a to nieje dobre.